jython/waitInput.py

```diff
@@ -1,247 +1,247 @@
-from javax.swing import *

```

Comment:
This file is originally written in ASCII (CRLF), but 2to3 changed into UTF-8 (LF). `style change` means this?

Reply:
> style change means this? 
Yes
> or should I keep this file as ASCII style?
I prefer this option, because I personally won't be using waitInput.py in the future and do not want to make unncessary changes to this file.